src/themes/classic/components/checkbox.js

```diff
@@ -0,0 +1,66 @@
+import { getPrimaryButtonStylesForIntent } from '../deprecated/helpers'
```

Comment:
If it's worth keeping, we can move it out of `/deprecated/*`

Reply:
Yeah - this is definitely worth keeping. I'll follow up since we use this in a few places.